server/index.js

```diff
@@ -3,10 +3,25 @@ let port = 3001;
 let app = express();
 const db = require('../database/index.js')
 
-app.use(express.static(__dirname+ '/../client/dist'));
 
-app.get('/0', (req, res) => {
-  db.getAll((err, data)=>{
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
+  next();

```

Comment:
theres a node module called cors that you can use to do this part; i think it would be cleaner